2023-12-08 14:42:39,177 INFO    MainThread:45103 [wandb_setup.py:_flush():71] setting env: {}
2023-12-08 14:42:39,177 INFO    MainThread:45103 [wandb_setup.py:_flush():71] setting login settings: {}
2023-12-08 14:42:39,177 INFO    MainThread:45103 [wandb_init.py:_log_setup():371] Logging user logs to /Users/thomas/PhD/research/code_papers/transfer_learning/cka_transfer_analysis/src/visualization/wandb/run-20231208_144239-13jhvktp/logs/debug.log
2023-12-08 14:42:39,177 INFO    MainThread:45103 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/thomas/PhD/research/code_papers/transfer_learning/cka_transfer_analysis/src/visualization/wandb/run-20231208_144239-13jhvktp/logs/debug-internal.log
2023-12-08 14:42:39,178 INFO    MainThread:45103 [wandb_init.py:init():404] calling init triggers
2023-12-08 14:42:39,178 INFO    MainThread:45103 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2023-12-08 14:42:39,178 INFO    MainThread:45103 [wandb_init.py:init():449] starting backend
2023-12-08 14:42:39,178 INFO    MainThread:45103 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-12-08 14:42:39,197 INFO    MainThread:45103 [backend.py:ensure_launched():199] starting backend process...
2023-12-08 14:42:39,203 INFO    MainThread:45103 [backend.py:ensure_launched():205] started backend process with pid: 45112
2023-12-08 14:42:39,206 INFO    MainThread:45103 [wandb_init.py:init():458] backend started and connected
2023-12-08 14:42:39,208 INFO    MainThread:45103 [wandb_init.py:init():519] updated telemetry
2023-12-08 14:42:39,226 INFO    MainThread:45103 [wandb_init.py:init():549] communicating current version
2023-12-08 14:42:40,549 INFO    MainThread:45103 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.16.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-12-08 14:42:40,549 INFO    MainThread:45103 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2023-12-08 14:42:41,317 INFO    MainThread:45103 [wandb_init.py:init():592] starting run threads in backend
2023-12-08 14:42:46,328 INFO    MainThread:45103 [wandb_run.py:_console_start():1816] atexit reg
2023-12-08 14:42:46,329 INFO    MainThread:45103 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2023-12-08 14:42:46,329 INFO    MainThread:45103 [wandb_run.py:_redirect():1695] Redirecting console.
2023-12-08 14:42:46,331 INFO    MainThread:45103 [wandb_run.py:_redirect():1751] Redirects installed.
2023-12-08 14:42:46,331 INFO    MainThread:45103 [wandb_init.py:init():619] run started, returning control to user process
2023-12-08 14:42:46,332 INFO    MainThread:45103 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2023-12-08 14:42:46,332 INFO    MainThread:45103 [wandb_run.py:_restore():1758] restore
2023-12-08 14:42:48,349 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 843
  total_bytes: 843
}

2023-12-08 14:42:48,634 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 843
  total_bytes: 843
}

2023-12-08 14:42:49,310 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 843
  total_bytes: 3068
}

2023-12-08 14:42:49,417 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 843
  total_bytes: 3068
}

2023-12-08 14:42:49,522 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 843
  total_bytes: 3068
}

2023-12-08 14:42:49,628 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 843
  total_bytes: 3068
}

2023-12-08 14:42:49,734 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3042
  total_bytes: 3068
}

2023-12-08 14:42:49,840 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}

2023-12-08 14:42:49,946 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}

2023-12-08 14:42:50,053 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}

2023-12-08 14:42:50,159 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}

2023-12-08 14:42:50,265 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}

2023-12-08 14:42:50,669 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}

2023-12-08 14:42:51,232 INFO    MainThread:45103 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3068
  total_bytes: 3068
}
local_info {
}

2023-12-08 14:42:52,676 INFO    MainThread:45103 [wandb_run.py:_append_files():2184] logging synced files
